### PR TITLE
Create secrets in quarkus-azure-services

### PR DESCRIPTION
**Pull request by gastaldi**
@majguo here is an example of how to create environment secrets in the repository 

- As requested in https://github.com/quarkiverse/quarkus-azure-services/pull/108

### CODE REVIEW

File: quarkus-azure-services.tf [hunk 1]
```diff
@@ -32,3 +32,16 @@ resource "github_team_membership" "quarkus_azure_services" {
...
+resource "github_actions_environment_secret" "quarkus_azure_services" {
+  for_each = {
+    "AZURE_CLIENT_ID"       = "foo"
+    "AZURE_TENANT_ID"       = "bar"
+    "AZURE_SUBSCRIPTION_ID" = "blah"
+  }
+  repository      = github_repository.quarkus_azure_services.name
+  environment     = "ci"
+  secret_name     = each.key
+  plaintext_value = each.value
```
Comment (gastaldi) [inline quarkus-azure-services.tf:46]: Do you need them to be encrypted? Just change to `encrypted_value` and generate teh encrypted value using https://docs.github.com/en/rest/actions/secrets?apiVersion=2022-11-28#create-or-update-an-environment-secret (I can come up with a script for that if necessary)

Comment (majguo) [reply quarkus-azure-services.tf:46]: Thank you @gastaldi. They needn't be encrypted as there are no **secrets** for these values. I just pushed a commit to update them with valid values.

**Review by majguo — APPROVED** [review]

**Comment by gastaldi** [comment]
Hm, if they are not secrets, perhaps we should create them as `Environment Variables` instead? WDYT?

**Comment by majguo** [comment]
@gastaldi Yeah, it's a good idea. Besides, I tested that `Environment Secrets` we added before are not retrieved by workflow run, not sure why as it works in my fork. Anyway, let's try to create them as `Environment Variables` as you suggested?

**Pull request merged**